app/components/c-panel.vue

```diff
@@ -1,9 +1,6 @@
 <template>
   <div :class="b(panelModifiers)">
-    <!-- TODO replace with e-heading -->
-    <div v-if="heading" :class="b('heading', headingModifiers)">
-      <span :class="b('heading-underline')">{{ heading }}</span>
-    </div>
+    <e-heading v-if="heading" :class="b('heading', headingModifiers)" tag-name="h2" underline>{{ heading }}</e-heading>
```

Comment:
- [ ] Don't use additional BEM classes on included components. Use a wrapper or properties of component itself.

By default it is recommended to add a wrapper for each placed component anyway.